apps/functions/applications-migration/common/handle-migration-with-response.js

```diff
@@ -3,23 +3,27 @@ import { validateMigration } from './validate-migration.js';
 /**
  * Wrapper function for migration functions that handles error handling and sends useful responses
  * @param {import('@azure/functions').Context} context
- * @param {string[]} caseReferences
+ * @param {string | string[]} caseReferences
  * @param {Function} migrationFunction
  * @param {string} entityName
+ * @param {boolean} allowCaseReferencesArray
  */
 export const handleMigrationWithResponse = async (
 	context,
 	caseReferences,
 	migrationFunction,
-	entityName
+	entityName,
+	allowCaseReferencesArray = false
 ) => {
-	if (!caseReferences || caseReferences.length === 0) {
+	const validationError = validateRequest(caseReferences, allowCaseReferencesArray);
+	if (validationError) {
```

Comment:
Wouldn't this always be true since we are returning an object

Reply:
it'll be undefined if it doesn't fall into any of the `if` blocks